### PR TITLE
v0.1.0 

### PR DESCRIPTION
**Pull request by AgentIsaacson**

### CODE REVIEW

File: Assets/-Game-/Entities/Enemies/Enemy.cs [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+				setVelocity.x = moveSpeed;
+				break;
+		}
+		rb.velocity = setVelocity;
+	}
+
+	void CheckForSwitchDirection(Collision2D other)
+	{
+		float yTop = transform.position.y + (transform.localScale.y / 2.1f);
+		float yBottom = transform.position.y - (transform.localScale.y / 2.1f);
```
Comment (AgentIsaacson) [inline Assets/-Game-/Entities/Enemies/Enemy.cs]: Why are these two things hard-coded at 2.1f? Can we make this relative?

File: Assets/-Game-/Entities/Enemies/Enemy.cs [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+		if (player != null)
+		{
+			if (other.gameObject == player.pastPlayer)
+			{
+
+				if (player.pastIsActivePlayer)
+				{
+					Debug.Log("Past Player");
+					player.KnockBack((other.transform.position - transform.position) * 300);
```
Comment (AgentIsaacson) [inline Assets/-Game-/Entities/Enemies/Enemy.cs]: Can we make the knockback factor a variable? Lines 109, 120, and 131

File: Assets/-Game-/Entities/Player/PlayerTakeDamage.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerTakeDamage : MonoBehaviour
```
Comment (AgentIsaacson) [inline Assets/-Game-/Entities/Player/PlayerTakeDamage.cs]: Why does this file exist? Can we delete it if there's no reason?

Comment (nateonline) [reply Assets/-Game-/Entities/Player/PlayerTakeDamage.cs]: Thanks, deleted!

File: Assets/-Game-/Entities/Enemies/Enemy.cs (hunk 2 above)
Comment (nateonline) [reply Assets/-Game-/Entities/Enemies/Enemy.cs]: Did!

File: Assets/-Game-/Entities/Enemies/Enemy.cs (hunk 1 above)
Comment (nateonline) [reply Assets/-Game-/Entities/Enemies/Enemy.cs]: Left a comment to explain!

**Pull request merged**